packages/examples/src/day2.ts

```diff
@@ -32,7 +32,10 @@ export const uischema = {
   elements: [
     {
       type: 'Control',
-      scope: '#/properties/name'
+      scope: '#/properties/name',
+      options: {
+        placeholder: 'Placeholder for name'
+      }
```

Comment:
We should not modify the `day2` example. It is referred to [by a blog series](https://eclipsesource.com/blogs/2016/12/27/json-forms-day-2-introducing-the-ui-schema/).